src/terms.ml

```diff
@@ -10,6 +10,98 @@
 open Extra
 open Timed
 
+(** {4 Constructors for decision trees} *)
+
+(** A constructor is the representation of a symbol along with the number of
+    arguments to which it is applied. *)
+type constructor = { c_mod : string list
```

Comment:
"constructor" is a very generic expression which is not very informative and could be confused with some usual notions like the constructors of inductive types or undefined symbols in rewriting theory. Please, find another name.

Add also a comment explaining what is c_mod.

Reply:
This type and the following modules seem independent of the type of terms.ml. They should be in a distinct file.

Reply:
What are those strings? Do they come from? Are they fully qualified identifiers?

Reply:
I think I will move this to a new file

Reply:
moved to a file treecons.ml